test/support/test_config.h

```diff
@@ -326,4 +326,28 @@
 // Drop view throws exceptions in libstdc++ 10
 #define _PSTL_LIBSTDCXX_XPU_DROP_VIEW_BROKEN (_GLIBCXX_RELEASE == 10)
 
+// std::ranges::view concept in libstdc++ 10 is broken, and in GCC 11 is broken prior to GCC 11.4 (i.e., __GLIBCXX__ < 20230714)
+#if defined(_GLIBCXX_RELEASE) && defined(__GLIBCXX__)
+#    define TEST_GCC10_STD_RANGES_VIEW_CONCEPTS_BROKEN                                                                 \
+        ((_GLIBCXX_RELEASE == 10) || (_GLIBCXX_RELEASE == 11 && __GLIBCXX__ < 20230714))
+#else
+#    define TEST_GCC10_STD_RANGES_VIEW_CONCEPTS_BROKEN 0
+#endif
+
+// std::ranges::views::all in libstdc++ 10 is broken prior to GCC 12.1
+#if defined(_GLIBCXX_RELEASE) && defined(__GLIBCXX__)
+#    define TEST_GCC10_STD_RANGES_VIEW_ALL_BROKEN                                                                      \

```

Comment:
Let's correct the name to clarify that it is a library (libstdc++) and not a compiler (g++) bug. It also affects multiple library versions. 

TEST_GCC10_STD_RANGES_VIEW_ALL_BROKEN -> TEST_LIBCXX_VIEW_ALL_BROKEN